src/components/Menu_Management/Zmones_Tab/PeopleListComponents/Popups/AssignTravelPopup.jsx

```diff
@@ -14,7 +14,11 @@ const AssignTravelPopup = props => {
         <Modal.Title> 
           <h4>
             Selected person: &nbsp; <b><i> {userInfo.name} {userInfo.surname} </i></b> 
-            <img src={userInfo.pic} alt="Profile pic" height="42" width="42" style={{borderRadius: "50%", marginLeft: "24px"}}/> 
+            <img 
+              src={ userInfo.pic == null ? 'https://www.w3schools.com/howto/img_avatar.png'
+                  : userInfo.pic == 'string' ? 'https://www.w3schools.com/howto/img_avatar2.png' 
+                  : userInfo.pic }
```

Comment:
Nested ternary conditionals tend to be frowned upon but I guess it's okay

Reply:
Tried to adapt code with current data and avoid blank profile pics (1 person has null, 5 have 'string' values in that field)

Reply:
I mean, the logic is fine it's just that in these cases you can use either if or case/switch statements

Works fine though